crates/ln-dlc-node/src/node/dlc_channel.rs

```diff
@@ -75,6 +75,24 @@ impl Node {
         Ok(())
     }
 
+    pub fn reject_dlc_channel_offer(&self, channel_id: &[u8; 32]) -> Result<()> {
+        let channel_id_hex = hex::encode(channel_id);
+
+        tracing::info!(channel_id = %channel_id_hex, "Rejecting DLC channel offer");
+
+        let (node_id, accept_sub_channel) = self
+            .sub_channel_manager
+            .accept_sub_channel(channel_id)
+            .map_err(|e| anyhow!(e.to_string()))?;
+
+        self.dlc_message_handler.send_message(
+            node_id,
+            Message::SubChannel(SubChannelMessage::Accept(accept_sub_channel)),
```

Comment:
🤔 This looks strange. Is `Accept` really the sub channel message that we are supposed to send here? 😅

Reply:
Oh, good catch. I was investigating if there was a way to reject subchannel open/close offers (there is none currently) and left this hanging around by mistake. It's not used and it's obviously wrong, so I'll remove it.